examples/data-centric/introduction/01-introduction-to-pygrid.ipynb

```diff
@@ -0,0 +1,741 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example (see illustration above), we have Andrew holding a piece of sensitive personal data, his special number 5. He can anonymize his data by decomposing the number 5 into two or more different numbers. Imagine that Andrew decides to decompose the number 5 into 2 and 3. This way, he can share his anonymized data with his friends Marianne and Bob.\n",
+    "\n",
+    "Neither Marianne nor Bob knows the real value of Andrew’s data because they each holds only a part of the special number. And since the decomposed numbers are still _alive_, either of them can perform any kind of operation without seeking the agreement of the other party. Indeed, while these numbers are encrypted between them, we’ll still be able to perform computations. __That way, we can use encrypted values to compute user’s data without showing any kind of sensitive information__."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![](images/3.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With these two concepts, let's now move on to PySyft and PyGrid."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## __Use Case: Private Statistical Analysis__\n",
+    "Let’s explore the following scenario with two work flows:\n",
+    "\n",
+    "1. A __data owner__ that wants to publish their sensitive data on their node. (In this case, a hospital pediatric ward).\n",
+    "2. A __data scientist__ that wants to find a specific dataset over the grid network to compute some statistical analysis.\n",
+    "\n",
+    "## __The Data Owner__\n",
+    "### __Step 1: Import PySyft and dependencies__\n",
+    "To publish the sensitive yet valuable data, the data owner first imports PyGrid and PySyft dependencies.\n",
+    "\n",
+    "In this example, we’ll import `syft` and wrap the standard `torch` modules with `syft.TorchHook` to add the additional privacy-preserving features that PySyft offers. If you have not set up your computer, follow [this](https://github.com/OpenMined/PyGrid) to install the necessary dependencies.\n",
+    "\n",
+    "Also, please make sure [these](https://github.com/OpenMined/PyGrid/tree/dev/examples#running-the-examples) instructions have been followed.\n",
+    "\n",
+    "Once done, you should have the following components:\n",
+    "\n",
+    " - PyGrid Network      (http://localhost:5000)\n",
+    " - PyGrid Node Bob (http://localhost:3000)\n",
+    " - PyGrid Node Alice   (http://localhost:3001)\n",
+    " \n",
+    "Yours URL / ports might vary depending on your configuration, so please ensure you look at the output of `docker-compose up` before moving forward.\n",
+    "\n",
+    "Visit http://localhost:5000/connected-nodes to ensure all nodes are ready and setup! You should get a response like below for the standard setup:\n",
```

Comment:
```suggestion
    "Visit http://network:7000/connected-nodes to ensure all nodes are ready and setup! You should get a response like below for the standard setup:\n",
```

Reply:
I think that if you visit that address from your web browser, must be localhost.
The main rule is
* If the containers must talk to each other use custom hostname, i.e. "alice"
* If you want to visit something from your laptop, then, use localhost.

In any case, I will review this as well

Reply:
I tried not using `localhost` and it worked in my Chrome. But sure that's a safe rule of thumbs!